requirements/frozen_docs.py3.txt

```diff
@@ -265,6 +265,7 @@ tabulate==0.7.7
 tzlocal==1.5.1
 uritemplate==3.0.0
 urllib3==1.24.2
+varmeth==0.0.2
```

Comment:
@digitronik this package should also be added to `template_scanned_imports.py`, cf `miq requirement add varmeth`

Reply:
right